src/Compiler/pars.fsy

```diff
@@ -5562,10 +5562,22 @@ recdExprCore:
             None, l
        | _ -> raiseParseErrorAt (rhs parseState 2) (FSComp.SR.parsFieldBinding ()) }
 
+  /* handles case like { x }  */
```

Comment:
can we also cover error recovering `recover / ends_coming_soon_or_recover`  ?

Reply:
Will also work with `struct {| x |}` ?

Reply:
> Will also work with `struct {| x |}` ? 

yes